cmd/minishift/cmd/util/oc.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
This is very specific logic.

Util namespace only deals with shared flags, basic `VMExists` checks and environment variable handling.

Reply:
I think the logic should be this: Can this function should be in pkg , if not can it be part of specific cmd or generic to multiple cmds, if yes then it should be in cmdUtil